packages/sku/lib/runBin.mjs

```diff
@@ -13,7 +16,7 @@ const resolveBin = (packageName, binName) => {
       ? packageJson.bin
       : packageJson.bin[binName || packageName];
 
-  return require.resolve(path.join(packageName, binPath));
+  return import.meta.resolve(path.join(packageName, binPath));
```

Comment:
This returns a file URL, rather than a file path. I think passing it to [`fileUrlToPath`](https://nodejs.org/api/url.html#urlfileurltopathurl-options) should get it into the right form.

Reply:
@DanDroryAu what about this one?

Reply:
Wait... I may have messed this up. I thought I pushed a fix for this. Let me check my local.

Reply:
> Your branch is ahead of 'origin/v14-lib-runners-mjs' by 1 commit.

Classic.